packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -164,4 +161,8 @@ export interface GetMarketOrdersOpts {
      * Default: 1.25.
      */
     sampleDistributionBase: number;
+    /**
+     * Fees for each liquidity source, expressed in ETH wei.
```

Comment:
This should be in gas*gasPrice right since it is simply multiplied by the base unit amount for the `fillPenalty`? I.e not `240000` but `240000*1e9`.

So the 0x penalty for `USDC/DAI` is:
```
ethToMakerAssetRate 0.000000000266756647
fillPenalty 640215 = 2400000000000000 * 0.000000000266756647
```

Without the gwei multiplier the penalty is too small.

Reply:
Yeah, the gas price needs to be incorporated in these because the optimizer won't assume one. I'll update the comment to be more clear.